Tests/AblyChatTests/Mocks/MockRealtimeChannel.swift

```diff
@@ -1,20 +1,19 @@
 import Ably
 @testable import AblyChat
 
-final class MockRealtimeChannel: NSObject, InternalRealtimeChannelProtocol {
+final actor MockRealtimeChannel: InternalRealtimeChannelProtocol {
```

Comment:
Shouldn't everything that calls async cocoa operations be an actor? Hence all `Default*` objects?

Reply:
> Shouldn't everything that calls async cocoa operations be an actor?

Why?